modules/cnv-removing-interface-from-nodes.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_node_network/cnv-configuring-node-network-policy
+
+[id="cnv-removing-interface-from-nodes_{context}"]
+= Removing an interface from nodes
+
+Remove an interface from nodes by editing the `NodeNetworkConfigurationPolicy` object that declares the `state` of the interface to `absent`.
+
+[NOTE]
+====
+Deleting the Policy that added an interface will not change the configuration of the network policy on the node.
+Although a `NodeNetworkConfigurationPolicy` is an object in the cluster, it only represents a desired configuration. +
+Similarly, removing an interface does not delete the Policy.
```

Comment:
@yossisegev - Nice one, thank you. I'm reluctant to be more prescriptive than this because it'll come down to however the admin wants to maintain their Policies.